ui/pages/import-token/import-token.component.js

```diff
@@ -242,7 +277,7 @@ class ImportToken extends Component {
 
         break;
       default:
-        if (customAddress !== emptyAddr) {
+        if (customAddress !== emptyAddr && !addressIsEmpty) {
           this.attemptToAutoFillTokenParams(customAddress);
         }
```

Comment:
May be it will be useful 
```
const addressIsEmpty = customAddress !== emptyAddr && customAddress.length === 0;
```

Reply:
fixed here: https://github.com/MetaMask/metamask-extension/pull/13271/commits/bf875f338df92e9d47aad398f4991bd08154bf5b